woocommerce/woocommerce
# Introduce wc_is_webhook_valid_status

### All Submissions:






### Changes proposed in this Pull Request:
Webhook status updates via the API was failing to the wrong sanitization callback being used for it. This PR introduces a new function called wc_is_webhook_valid_status and makes use of that function to run as the sanitization callback.

Closes #22200 

### How to test the changes in this Pull Request:

1. Check that new unit tests pass.
2. Make an API call to update a webhook status, check that the status was updated.

### Other information:




### Changelog entry

> Fix - Run webhook status updates through new wc_is_webhook_valid_status functions when doing API requests.
